CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Change Log
 
-## Unreleased
+## 1.0.0 - _2020-06-11_
 
-_2020-06-11_
+- Rename `applyLayout()` to `layoutBy()`.
+- Add utility axis solvers:
+  - [`contourWidthMatchParent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L307) and [`contourHeightMatchParent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L336)
+  - [`contourWidthWrapContent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L328) and [`contourHeightWrapContent()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L357)
+  - [`matchXTo()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L564) and [`matchYTo()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L592)
+  - [`matchParentX()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L578) and [`matchParentY()`](https://github.com/cashapp/contour/blob/e45ab4a625dbf37ec419d864ef7692c3c7bb01c7/contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt#L606)
+  
+### Breaking changes
```

Comment:
Not a breaking change pre-1.0?

Reply:
No these were introduced after `0.1.7`: https://github.com/cashapp/contour/compare/0.1.7...master

Reply:
Correct, but any change introduced before 1.0 is not a breaking change.  So should we remove this header?

Reply:
It's breaking for users of the previous version. You'd call it out for a 0.8 release. 1.0 is the same as 0.8 here except that it comes with a future promise of compatibility and not just an indication of backwards incompatibility.

Reply:
Got it.  So it's ok to break, but the idea is to still call it out?  👍

Reply:
It may be worth it to put "from 0.1" in the title to be abundantly clear about what it refers to.

Reply:
Sure, renamed `### Breaking changes` to `### Breaking changes (for 0.1.7)`